api/resolvers/notifications.js

```diff
@@ -381,6 +390,12 @@ export default {
   TerritoryPost: {
     item: async (n, args, { models, me }) => getItem(n, { id: n.id }, { models, me })
   },
+  Reminder: {
+    item: async (n, args, { models, me }) => {
+      const { itemId } = await models.reminder.findUnique({ where: { id: Number(n.id) } })
+      return await getItem(n, { id: itemId }, { models, me })
+    }
```

Comment:
This isn't a horrible idea, though I'd probably just return null instead of throwing an error. I can make the change if the human reviewer thinks we should :)

Reply:
Bah this shouldn't get here if the reminder doesn't exist. Generally, we don't need to worry about read consistency if we make sure our writes are consistent.